cmd/influx_inspect/report/report.go

```diff
@@ -218,64 +219,6 @@ func sortKeys(vals map[string]counter) (keys []string) {
 	return keys
 }
 
-func (cmd *Command) isShardDir(dir string) error {
```

Comment:
These are extracted to `reporthelper`. Open to suggestions on a better name - `shardinspect`?